DataFormats/simulation/include/SimulationDataFormat/MCTrack.h

```diff
@@ -339,6 +348,8 @@ inline MCTrackT<T>::MCTrackT(const TParticle& part)
     setToBeDone(true); // if inhibited, it had to be done: restore flag
     setInhibited(true);
   }
+  // set MC generator status code only for primaries
+  mStatusCode = part.TestBit(ParticleStatus::kPrimary) ? part.GetStatusCode() : -1;
```

Comment:
@amorsch : Is `-1` ok as `not set` or should we use 0.

Reply:
@sawenzel 
The process type for secondaries can be obtained from 
part.GetUniqueID();

Reply:
I know. But the process is already encoded in MCTrack elsewhere (for now). I would leave the logic of mixing status codes and the process to the filling part in the AOD creator. It is better that users call `getStatusCode()` and `getProc()`, so that we can change the data model internally without causing any disruption.

Reply:
ok then -1 is ok.